.github/workflows/maven-v1.yml

```
@@ -0,0 +1,42 @@
+  push:
+    branches: [ "1.x" ]
+  pull_request:
+    branches: [ "1.x" ]
+
+jobs:
+  build-vaadin23:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up JDK
+      uses: actions/setup-java@v3
```

actionlint (1.7.9)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)